### PR TITLE
dispactching expect_table_row_count_to_equal

### PR DESCRIPTION
**Pull request by alieus**
dispactching `expect_table_row_count_to_equal` to create a shim to make it compatible with TSQL

### CODE REVIEW

**Review by clausherther — CHANGES_REQUESTED** [review]
Thanks for adding that, looks good! One small change: could we add spaces after commas in here, like so:
```
(model, value, row_condition=None)
```
Apologies if this isn't consistent elsewhere.

**Review by clausherther — CHANGES_REQUESTED** [review]
Sorry, actually, one more change:
We should not pass None in the dispatch, the main method already does this.
```
(model, value, row_condition)
```

**Comment by alieus** [comment]
> Thanks for adding that, looks good! One small change: could we add spaces after commas in here, like so:
> 
> ```
> (model, value, row_condition=None)
> ```
> 
> Apologies if this isn't consistent elsewhere.

**Comment by clausherther** [comment]
Hi @alieus! Anything else you need from me to make those changes?

**Comment by alieus** [comment]
Hey @clausherther. I believe I made the changes you requested. This should be good to merge. Let me know if you think otherwise.

File: macros/schema_tests/table_shape/expect_table_row_count_to_equal.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{%- macro test_expect_table_row_count_to_equal(model,value,row_condition=None) -%}
+{%- macro test_expect_table_row_count_to_equal(model, value, row_condition) -%}
     {{ adapter.dispatch('test_expect_table_row_count_to_equal', packages=dbt_expectations._get_namespaces()) (model,value,row_condition=None) }}
```
Comment (clausherther) [inline macros/schema_tests/table_shape/expect_table_row_count_to_equal.sql:2]: Hey! Sorry, small thing: could we add spaces here as well like you did above?
Also, please keep the `None` default in the macro parameters, but take it out of the dispatch call.
```
{%- macro test_expect_table_row_count_to_equal(model, value, row_condition=None) -%}    
    {{ adapter.dispatch('test_expect_table_row_count_to_equal', packages=dbt_expectations._get_namespaces()) (model, value, row_condition) }}
```

**Review by clausherther — CHANGES_REQUESTED** [review]

**Review by clausherther — APPROVED** [review]
LGTM, thanks!

**Pull request merged**